README.md

```diff
@@ -6,99 +6,157 @@
+[ONNX](https://github.com/onnx/onnx) is an interoperable format for machine learning models supported by various ML and DNN frameworks and tools. The universal format makes it easier to interoperate between frameworks and maximize the reach of hardware optimization investments.
+***
+**Setup**
+* [Installation](#installation)
+* [APIs and Official Binaries](#apis-and-official-builds)
+* [Building from Source](#building-from-source)
 
-# Why use ONNX Runtime
-ONNX Runtime has an open architecture that is continually evolving to address the newest developments and challenges in AI and Deep Learning. ONNX Runtime stays up to date with the ONNX standard, supporting all ONNX releases with future compatibility and maintaining backwards compatibility with prior releases.
+**Getting Started**
+* [Deploying ONNX Runtime](#deploying-onnx-runtime)
+* [Getting ONNX Models](#getting-onnx-models)
+* [Examples and Tutorials](#examples-and-tutorials)
 
-ONNX Runtime continuously strives to provide top performance for a broad and growing number of usage scenarios in Machine Learning. Our investments focus on:
-1. Run any ONNX model
-2. High performance
-3. Cross platform
+**About ONNX Runtime**
+* [Why use ONNX Runtime](#why-use-onnx-runtime)
+* [Design and Key Features](#design-and-key-features)
 
-## Run any ONNX model
+**[Contributions and Feedback](#contribute)**
 
-### Alignment with ONNX Releases
-ONNX Runtime provides comprehensive support of the ONNX spec and can be used to run all models based on ONNX v1.2.1 and higher. See ONNX version release details [here](https://github.com/onnx/onnx/releases).
+**[License](#license)**
+***
+## Installation
+**Quick Start:** The [ONNX-Ecosystem Docker container image](https://github.com/onnx/onnx-docker/tree/master/onnx-ecosystem) is available on Dockerhub and includes ONNX Runtime (CPU, Python), dependencies, tools to convert from various frameworks, and Jupyter notebooks to help get started.
 
-As of May 2019, ONNX Runtime supports ONNX 1.5 (opset10). See [this table](https://github.com/Microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix) for details on ONNX Runtime and ONNX versioning compatibility, 
+Additional dockerfiles for some features can be found [here](https://github.com/microsoft/onnxruntime/tree/master/dockerfiles).
 
-### Traditional ML support
-ONNX Runtime fully supports the ONNX-ML profile of the ONNX spec for traditional ML scenarios.
+### System Requirements
+* ONNX Runtime binaries in the CPU packages use OpenMP and depend on the library being available at runtime in the
+system.
+  * For Windows, OpenMP support comes as part of VC runtime. It is also available as redist packages:
+    [vc_redist.x64.exe](https://aka.ms/vs/15/release/vc_redist.x64.exe) and [vc_redist.x86.exe](https://aka.ms/vs/15/release/vc_redist.x86.exe)
+  * For Linux, the system must have the libgomp.so.1 which can be installed using ```apt-get install libgomp1```.
```

Comment:
Surround libgomp.so.1 with ` so it matches the formatting as in the following command